src/utils/CalciteScrim.tsx

```diff
@@ -0,0 +1,27 @@
+import { FunctionalComponent, h } from "@stencil/core";
+const divStyle = {
+  alignItems: "center",
+  animation:
+    "calcite-app-fade-in var(--calcite-app-animation-time) var(--calcite-app-easing-function)",
```

Comment:
I'm not sure these vars will work in `calcite-components`. Do we need to consider making the scrim an undocumented `calcite-component` that can be used in both projects? @asangma @jcfranco